qtwidgetsextra/groupbox/QAbstractButtonGroupBox_p.h

```diff
@@ -0,0 +1,66 @@
+#ifndef QABSTRACTBUTTONGROUPBOX_P_H
+#define QABSTRACTBUTTONGROUPBOX_P_H
+
+#include <QAbstractButton>
+#include <QButtonGroup>
+#include <QBoxLayout>
+
+class QAbstractButtonGroupBox;
+
+class QAbstractButtonGroupBoxPrivate : public QObject
+{
+    Q_OBJECT
+public:
+    QAbstractButtonGroupBoxPrivate(QAbstractButtonGroupBox *widgetP);
+
+    void clear() const;
+    void setOrientation(Qt::Orientation newOrientation);
+    QStringList items() const;
+    void removeItem(int index);
+
+    void insertButton(int index, QAbstractButton *button, int buttonId = -1);
+    void addButton(QAbstractButton *button, int buttonId = -1);
+
+    void setCheckable(bool checkable);
+
+    void setChecked(bool checked, int index);
+    bool isChecked(int index);
+
+    void setEnabled(bool enabled);
+    bool isEnabled(int index);
+    void setEnabled(int index, bool enabled);
+
+    int index(int buttonId);
+
+    void setButtonId(int index, int buttonId);
+    int buttonId(int index);
+
+    int count();
+    bool isEmpty();
+
+private slots:
```

Comment:
It's private class - let put all public, no need of private / protected etc here